### ISSUE TITLE
Possible BP and ProjectE/ProjectEX issue.

### ISSUE THREAD

**Issue by border999** [Johni0702/BetterPortals#474]
https://gist.github.com/border999/a4d321c7b4026cfcf6e14de7ddb48f1d

**Comment by border999** [comment]
I'll also not that I don't attach the full log because mod makers have started to hate me for that because my logs are very large.

**Comment by Johni0702** [comment]
Does this have any noticeable negative effect?
From looking at ProjectE's source, it looks like it'd be safe to ignore the warning. So, if you don't notice any bugs (specifically with ProjectE's JEI integration, that's what the code in question is doing), then I'm going to add a special case for it to be ignored (which will end up in the next release, whenever there's something pressing to fix).

**Comment by border999** [comment]
no, that seems fair enough, assuming I don't find any glaring issues with better portals this should be it.
